boards/xtensa/up_squared_adsp/tools/fw_loader/driver.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import fcntl
+import struct
+import mmap
+import time
+import logging
+
+from ctypes import cast, POINTER, c_uint8, c_uint32, c_uint16, c_uint64, addressof, byref, sizeof
+import registers as regs_def
+
+
+# diag_driver file
+DIAG_DRV_PATH = "/dev/hda"
+
+# Diag Driver Definition - sof-diagnostic-driver/ioctl.h
+
+# IOCTL definition
+CMD_OPEN_DEVICE     = 0x47
+CMD_ALLOC_MEMORY    = 0x3A
+CMD_FREE_MEMORY     = 0x3B
+
+CMD_OPEN_DEVICE_LEN = 40
+
```

Comment:
I think python requires 2 blank lines here and in all other places https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Hum.. I checked the reference and spec, but I couldn't find for double space for constant definition. Anyway, I went over the code to align the style as much as possible.

Reply:
I meant double space before class definition